Makefile

```diff
@@ -265,7 +265,7 @@ ifeq ($(PYTEST),)
 	exit 1
 else
 # Explicitly hand DEVELOPER and VALGRIND so you can override on make cmd line.
-	PYTHONPATH=`pwd`/contrib/pyln-client:`pwd`/contrib/pyln-testing:`pwd`/contrib/pylightning:$$PYTHONPATH TEST_DEBUG=1 DEVELOPER=$(DEVELOPER) VALGRIND=$(VALGRIND) $(PYTEST) tests/ $(PYTEST_OPTS)
+	PYTHONPATH=`pwd`/contrib/pyln-client:`pwd`/contrib/pyln-testing:`pwd`/contrib/pylightning:`pwd`/contrib/pyln-proto/:$(PYTHONPATH) TEST_DEBUG=1 DEVELOPER=$(DEVELOPER) VALGRIND=$(VALGRIND) $(PYTEST) tests/ $(PYTEST_OPTS)
```

Comment:
Seems like `Makefile`s are special with the syntax they allow variable references vs. subshell execution. They have very different semantics: the backticks execute the `pwd` command, while `$(PYTHONPATH)` interpolate existing variables. We could define a working directory `CWD` and use that with `$(CWD)` if that works better. Executing multiple `pwd` just to build a colon-separated path does seem a bit strange now that you mention it.

Reply:
I think $$(cmd) == \`cmd\` ?

Reply:
I prefer backticks, because I'm old.  But they don't nest neatly, so $( became a thing.  Of course, that's `$$(` to get through a Makefile.

There's nothing wrong with `, despite shellcheck's opinion :)